mstm/__init__.py

```diff
@@ -40,9 +40,11 @@
 import os
 import shutil
 import numpy as np
+from scipy import interpolate
+from scipy import integrate
 
 # change to match the filename of your executable
-MSTM_EXE = 'mstm.exe'
+MSTM_EXE = 'mstm_ubuntu.exe'
```

Comment:
also, by "user's path" I mean the environment variable that lists all the directories an executable file can be in.  Usually this includes the current path as well as a bunch of other directories the user may have added.  So I probably should have said "user's PATH"